sklearn/ensemble/tests/test_iforest.py

```diff
@@ -53,9 +53,9 @@ def test_iforest_sparse(global_random_seed):
     X_train, X_test = train_test_split(diabetes.data[:50], random_state=rng)
     grid = ParameterGrid({"max_samples": [0.5, 1.0], "bootstrap": [True, False]})
 
-    for sparse_format in [csc_matrix, csr_matrix]:
-        X_train_sparse = sparse_format(X_train)
-        X_test_sparse = sparse_format(X_test)
+    for sparse_container in [*CSC_CONTAINERS, *CSR_CONTAINERS]:
```

Comment:
I think you can remove this loop and instead parametrize using `[*CSC_CONTAINERS, *CSR_CONTAINERS]`.

Reply:
Thanks, not sure why I missed that lol.